src/common/code_utils.hpp

```diff
@@ -165,6 +169,14 @@ template <typename T, typename... Args> std::unique_ptr<T> MakeUnique(Args &&...
     return std::unique_ptr<T>(new T(std::forward<Args>(args)...));
 }
 
+/**
+ * This method converts 8 uint8_t bytes into uint64_t.
```

Comment:
Document whether it is big-endian or little endian?

Reply:
Added document "using big-endian". Thanks for the suggestion.